minimal/compose/nginx/production/nginx.conf

```diff
@@ -33,11 +33,27 @@ http {
     }
 
     server {
-        listen 80;
-        charset     utf-8;
+        listen 80 default_server;
+        listen [::]:80 default_server;
+        server_name ${DOMAIN};
+        return 301 https://$server_name$request_uri;
+    }
+
+    server {
+        # https://www.digitalocean.com/community/tutorials/how-to-create-a-self-signed-ssl-certificate-for-nginx-in-ubuntu-16-04
```

Comment:
We can loose this line. No need for it here since we won't be using self-signed certificates.